### PR TITLE
SEO-3260-PHP-Description-Too-Short

### PR DESCRIPTION
**Pull request by berylouma**
Hi @Aishwarya-Ganesan 

I have worked on the descriptions.

Regards,
Beryl

### CODE REVIEW

File: PHP/Button/Getting-Started.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Getting-Started
-description: getting started
+title: Getting Started | JavaScript | Button | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline PHP/Button/Getting-Started.md:3]: Change platform `PHP`

File: PHP/Button/Getting-Started.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+description: Learn here more about getting started with Syncfusion Essential JavaScript Button Control, its elements, and more.
```
Comment (gowthamanthavasiyappan) [inline PHP/Button/Getting-Started.md:4]: Change this place also

File: PHP/Button/Getting-Started.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 platform: js
 control: Button
 documentation: ug
 ---
 
-# Getting Started
+# Getting Started with JavaScript Button
```
Comment (gowthamanthavasiyappan) [inline PHP/Button/Getting-Started.md:10]: Check and change in all places [change PHP instead of JavaScript]

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

File: PHP/Button/Getting-Started.md (hunk 1 above)
Comment (berylouma) [reply PHP/Button/Getting-Started.md:3]: Done

File: PHP/Button/Getting-Started.md (hunk 2 above)
Comment (berylouma) [reply PHP/Button/Getting-Started.md:4]: Done

File: PHP/Button/Getting-Started.md (hunk 3 above)
Comment (berylouma) [reply PHP/Button/Getting-Started.md:10]: Done

**Pull request closed without merging**